cinema_service/settings.py

```diff
@@ -12,6 +12,10 @@
 
 from pathlib import Path
 
+from rest_framework import authentication
+
+from user.permissions import IsAdminOrIfAuthenticatedReadOnly
```

Comment:
No need for these two imports - you enter default permission / auth classes as string and django will resolve them